geoapi/tests/conftest.py

```diff
@@ -121,33 +120,22 @@ def public_projects_fixture(projects_fixture):
 
 
 @pytest.fixture(scope="function")
-def observable_projects_fixture():
+def watch_content_users_projects_fixture():
     u1 = db_session.query(User).filter(User.username == "test1").first()
     proj = Project(name="test_observable",
```

Comment:
Pedantic comment 10: For consistency, wouldn't it be better to use `project` instead of `proj`?

Reply:
updated in f2a4f84325346bc88dc6169f1cdf7c9023b11a5c